models/breast_density_classification/configs/metadata.json

```diff
@@ -1,19 +1,20 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.1.5",
+    "version": "0.1.6",
     "changelog": {
+        "0.1.6": "Remove meta dict usage",
         "0.1.5": "Fixed duplication of input output format section",
         "0.1.4": "Changed Readme",
         "0.1.3": "Change input_dim from 229 to 299",
         "0.1.2": "black autofix format and add name tag",
         "0.1.1": "update license files",
         "0.1.0": "complete the model package"
     },
-    "monai_version": "1.0.0",
-    "pytorch_version": "1.12.1",
-    "numpy_version": "1.21.2",
+    "monai_version": "865972f7a791bf7b42efbcd87c8402bd865b329e",
```

Comment:
Hi @KumoLiu @wyli , do you think we should hold the PR until we have a new monai tag that covers this commit?

Reply:
yes, this is for temp test only